woeip/apps/air_quality/views.py

```diff
@@ -1,61 +1,40 @@
-def upload(request):
-    """Upload data for a session collected using the Dustrak air quality device and a separate GPS
-    log file.
-    """
-    request_user = request.user
-    if request.method == 'POST':
-        form = forms.DustrakSessionForm(request.POST, request.FILES)
-        form_instance = form.instance
+class Upload(LoginRequiredMixin, View):
+    def get(self, request):
+        """Present file upload form to user"""
+        return render(self.request, 'air_quality/upload.html', {
+            'form': SessionDataForm
+        })
+
+    def post(self, request):
+        """Save files to SessionData table"""
+        files = self.request.FILES
+        form = SessionDataForm(self.request.POST, files)
         if form.is_valid():
             form.save()
-            try:
-                air_sensor = models.Sensor.objects.get(name='Dustrak')
-                gps_sensor = models.Sensor.objects.get(name='GPS')
-
-            except (ObjectDoesNotExist) as e:
-                messages.add_message(request, messages.ERROR, f'File upload failed, error: {e}')
-                logger.exception('Could not find sensor information')
-                return redirect('upload')
-
-            air_quality = models.SessionData(upload=request.FILES['air_quality'],
-                                             sensor=air_sensor,
-                                             session=form_instance,
-                                             uploaded_by=request_user)
-            gps = models.SessionData(upload=request.FILES['gps'],
-                                     sensor=gps_sensor,
-                                     session=form_instance,
-                                     uploaded_by=request_user)
-
-            air_quality_contents = force_text(request.FILES['air_quality'].read())
-            _, air_quality_data = dustrak.load_dustrak(air_quality_contents, form.data['timezone'])
-
-            gps_contents = force_text(request.FILES['gps'].read())
-            gps_data = dustrak.load_gps(gps_contents)
-            joined_data = dustrak.join(air_quality_data, gps_data)
-
-            air_quality.save()
-            gps.save()
-            dustrak.save(joined_data, form_instance)
-
-            messages.add_message(request, messages.SUCCESS, 'Files successfully uploaded')
-            return redirect('upload')
-
-    else:
-        form = forms.DustrakSessionForm(initial={'collected_by': request_user})
-
-    return render(request, 'air_quality/upload.html', {
-        'user': request_user, 'form': form, 'upload_page': 'active'})
+            path = redirect('view_session_data')
+        else:
+            messages.add_message(self.request, messages.ERROR, 'File upload error')
+            path = redirect('upload')
+        return path
+
+
+class ViewSessionData(View):
+    """Provide temporary development page to view all uploaded SessionDatas."""
+    def get(self, request):
+        sessionData_list = SessionData.objects.all()
```

Comment:
please use a consistent case (snake case for python variables), in this instance `session_data_list`.

Reply:
Yeah... I figured it was too weird. I'll be normal and change it.